tests/browser/snapshots/storybook-docs/Docs-Components-IdleTimeout-Docs-prop-table-matches-snapshot-1.txt

```diff
@@ -2,12 +2,12 @@
   [
     "continueSessionText",
     "The text for the 'continue session' button in warning dialog.\nReactNode",
-    "Continue session"
+    "t('idleTimeoutDialog.continueSessionButtonText')"
   ],
   [
     "endSessionButtonText",
     "The text for the button that ends the session in warning dialog.\nReactNode",
-    "Logout"
+    "t('idleTimeoutDialog.endSessionButtonText')"
```

Comment:
Huh, I'm not sure if this is more helpful or less. On the one hand, it shows them that it's dynamic. On the other, this is a useless string (but they can find out what it actually is through playing with the story)